### PR TITLE
Spatially assign geo boundaries

### PR DESCRIPTION
**Pull request by baolingz**

### CODE REVIEW

File: lib/zip_url_to_postgis.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def url_to_postgis(url, table_name):
         geometryType = 'Multipolygon',
         layerCreationOptions = ['precision=NO'],
         format='PostgreSQL',
-        srcSRS='EPSG:2263', 
+        srcSRS='EPSG:4326',
```
Comment (SPTKL) [inline lib/zip_url_to_postgis.py]: it should be 2263

File: sql/assign_geo_boundaries.sql [hunk 2]
```diff
@@ -0,0 +1,58 @@
+-- Spatially assign geo attributes where record has geometry but the geom bound info is NULL
+
+-- community district
+UPDATE facilities_1 a
```
Comment (SPTKL) [inline sql/assign_geo_boundaries.sql]: change `facilities_1` to `facilities`

Comment (baolingz) [reply sql/assign_geo_boundaries.sql]: fixed

File: lib/zip_url_to_postgis.py (hunk 1 above)
Comment (baolingz) [reply lib/zip_url_to_postgis.py]: `doitt_zipcodeboundaries` cannot spatially join with `facilities` successfully when it's 2263

Comment (SPTKL) [reply lib/zip_url_to_postgis.py]: ok

**Pull request merged**